### ISSUE TITLE
Additional GeoSPARQL requirements from Loc-I project

### ISSUE THREAD

**Issue by dr-shorthair** [opengeospatial/geosemantics-dwg#73]
We have been adding features to a 'GeoSPARQL Extensions' ontology that are required for a significant deployment in Australia. See https://github.com/CSIRO-enviro-informatics/geosparql-ext-ont 

This includes 
- predicates for length, area, volume
- predicate for centroid
- inverse property for `geo:hasGeometry`
- explicit link to CRS description
- predicates for spatial resolution

**Comment by FransKnibbe** [comment]
@dr-shorthair: Do you think everything added by the extension should be adopted by the core GeoSPARQL ontology? Or are there benefits to the extension being an extension (for example, you having more control over it)?
The process leading to a possible next version of GeoSPARQL has been to file change requests in the OGC StandardsTracker (see http://ogc.standardstracker.org/requestlist.cgi?quicksearch=geosparql). One easy option would be to add a request to migrate additions from the extensions ontology to the GeoSPARQL ontology. There is bound to be overlap with other change requests, but comparing them all for (partial) duplicates may be too much work at this stage (it hasn't been decided yet that work on a next version of GeoSPARQL will go ahead).

**Comment by dr-shorthair** [comment]
Not necessarily. I'm just putting it on the table. These are properties that are required for our application (actually length and volume not yet, but they were obvious once we were looking at area). They are rather minor additions mostly.

**Comment by FransKnibbe** [comment]
I see geox is mentioned in the draft of the charter for an update of GeoSPARQL, so even if it doesn't translate to an issue in the tracker, it should not remain under the radar.
